xbmc/video/VideoDatabase.cpp

```diff
@@ -3571,6 +3599,87 @@ bool CVideoDatabase::GetStreamDetails(CVideoInfoTag& tag) const
   return retVal;
 }
  
+bool CVideoDatabase::GetStreamDetails(const std::string &media_type, std::map<int, CVideoInfoTag>& details)
```

Comment:
There's a `MediaType` typedef which should be used instead of `std::string` to make it more obvious that it should be one of the supported `MediaTypeFoo` values.